tfx/components/evaluator/executor.py

```diff
@@ -213,9 +235,7 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
              slice_spec=slice_spec,
              tensor_adapter_config=tensor_adapter_config))
       else:
-        data = pipeline | 'ReadFromTFRecord' >> beam.io.ReadFromTFRecord(
-            file_pattern=file_pattern)
-        (data
+        (examples_list | 'FlattenExamples' >> beam.Flatten()
```

Comment:
this seems incorrect, examples_list is empty in this else branch (where is beam.io.ReadFromTFRecord?)

Reply:
There's no else branch here anymore. beam.io.ReadFromTFRecord is in line 199.